src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyCSVTest.java

```diff
@@ -118,6 +134,51 @@ public void testCSVFromURL() throws SQLException {
         }
     }
 
+    @Test
+    @DisplayName("Test setEscapeColumnDelimitersCSV")
+    public void testEscapeColumnDelimitersCSV() throws SQLException, FileNotFoundException, IOException {
+        String tableName = AbstractSQLGenerator.escapeIdentifier(RandomUtil.getIdentifier("BulkEscape"));
+        String fileName = filePath + inputFileDelimiterEscape;
+        /*
+         * The list below is the copy of inputFileDelimiterEscape with quotes removed.
+         */
+        String[][] expectedEscaped = new String[5][4];
+        expectedEscaped[0] = new String[] {"test", " test\"", "no,split", " testNoQuote"};
+        expectedEscaped[1] = new String[] {null, null, null, null};
+        expectedEscaped[2] = new String[] {"\"", "test\"test", "test,\"  test", null};
+        expectedEscaped[3] = new String[] {"testNoQuote  ", " testSpaceAround ", " testSpaceInside ",
+                "  testSpaceQuote\" "};
+        expectedEscaped[4] = new String[] {null, null, null, " testSpaceInside "};
+
+        try (Connection con = getConnection(); Statement stmt = con.createStatement();
+                SQLServerBulkCopy bulkCopy = new SQLServerBulkCopy(con);
+                SQLServerBulkCSVFileRecord fileRecord = new SQLServerBulkCSVFileRecord(fileName, encoding, delimiter,
+                        false)) {
+            bulkCopy.setDestinationTableName(tableName);
+            fileRecord.setEscapeColumnDelimitersCSV(true);
+            fileRecord.addColumnMetadata(1, null, java.sql.Types.INTEGER, 0, 0);
+            fileRecord.addColumnMetadata(2, null, java.sql.Types.VARCHAR, 50, 0);
+            fileRecord.addColumnMetadata(3, null, java.sql.Types.VARCHAR, 50, 0);
+            fileRecord.addColumnMetadata(4, null, java.sql.Types.VARCHAR, 50, 0);
+            fileRecord.addColumnMetadata(5, null, java.sql.Types.VARCHAR, 50, 0);
+            stmt.executeUpdate(
```

Comment:
My formatter shows that this line needs to be formatted. Can you try re-formatting this file?

Reply:
You are right.